meson.build

```diff
@@ -1264,7 +1264,8 @@ vulkan_opt = get_option('vulkan').require(
     error_message: 'libplacebo could not be found!',
 )
 vulkan = dependency('vulkan', required: vulkan_opt)
```

Comment:
would be better to just version check here

Reply:
Thanks for suggestion.  I checked and it looks like vulkan 1.1.70 introduced the symbol.  The problem that I have is that when cross compiling for android, mpv vulkan support builds but then at runtime it causes a fatal exception because the symbol never got linked properly.

Reply:
that would be an issue with android or its NDK

Reply:
before context.c was just refactored, mpv vulkan support in android did not link in any external vulkan symbols

Reply:
> mpv vulkan support builds but then at runtime it causes a fatal exception because the symbol never got linked properly.

I just tested this and it works fine for me (on Android 11). libmpv starts up correctly, it merely fails to initialize since my phone only has vk1.1

Reply:
This is what my error looks like on cm-14.1:

```
09-09 23:51:52.804 19403 19403 E AndroidRuntime: FATAL EXCEPTION: main
09-09 23:51:52.804 19403 19403 E AndroidRuntime: Process: is.xyz.mpv, PID: 19403
09-09 23:51:52.804 19403 19403 E AndroidRuntime: java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "vkGetPhysicalDeviceProperties2" referenced by "/data/app/is.xyz.mpv-1/base.apk!/lib/arm64-v8a/libmpv.so"...
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at java.lang.Runtime.loadLibrary0(Runtime.java:989)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at java.lang.System.loadLibrary(System.java:1562)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at is.xyz.mpv.MPVLib.<clinit>(MPVLib.java:19)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at is.xyz.mpv.MPVLib.create(Native Method)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at is.xyz.mpv.MPVView.initialize(MPVView.kt:18)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at is.xyz.mpv.MPVActivity.onCreate(MPVActivity.kt:304)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:6684)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1119)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2637)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2751)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at android.app.ActivityThread.-wrap12(ActivityThread.java)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1496)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:102)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:154)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6186)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
09-09 23:51:52.804 19403 19403 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
```

Reply:
According to this [document](https://source.android.com/docs/core/graphics/implement-vulkan#versions) you indeed need at least Android 9.
I don't think it makes a practical difference since libplacebo now requires vk1.2 anyway.

> I do not have vulkan support in mpv enabled

evidently you do. if you don't want vulkan, disable it